deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -292,5 +292,8 @@
 
 {{- $root := . -}}
 {{- range $path, $bytes := .Files.Glob "assets/operations/pre_render_scripts/api_*" }}
+{{- if and (not $root.Values.features.suse_buildpacks.enabled) (regexMatch "api_suse-.*?-buildpack" $path ) }}
```

Comment:
This is not very natural to read. How about we split that logic and make it clearer?

```
{{- if .Values.features.suse_buildpacks.enabled }}
  {{- range $path, $bytes := .Files.Glob "assets/operations/pre_render_scripts/api_*" }}
    {{- $root.Files.Get $path }}
  {{- end }}
{{- else }}
  {{- range $path, $bytes := .Files.Glob "assets/operations/pre_render_scripts/api_*" }}
    {{- if not (regexMatch "api_suse-.*?-buildpack" $path) }}
      {{- $root.Files.Get $path }}
    {{- end }}
  {{- end }}
{{- end }}
```

It's longer but easier to read.